org.gtk.oomox.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "org.gtk.oomox",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "oomox-gui",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=~/.themes",
+        "--filesystem=~/.icons",
+        "--socket=session-bus",
```

Comment:
Overly broad.

Reply:
should i specify `own-name` or what do you mean?

Reply:
No, your DBus name should match your app-id.